Src/FluentAssertions.Json/JTokenDifferentiator.cs

```diff
@@ -166,19 +166,71 @@ private static Difference CompareValues(JValue actual, JValue expected, JPath pa
             
             return null;
         }
+
+        private static string Describe(JTokenType jTokenType)
+        {
+            switch (jTokenType)
+            {
+                case JTokenType.None:
+                    return "type none";
+                case JTokenType.Object:
+                    return "an object";
+                case JTokenType.Array:
+                    return "an array";
+                case JTokenType.Constructor:
+                    return "a constructor";
+                case JTokenType.Property:
+                    return "a property";
+                case JTokenType.Comment:
+                    return "a comment";
+                case JTokenType.Integer:
+                    return "an integer";
+                case JTokenType.Float:
+                    return "a float";
+                case JTokenType.String:
+                    return "a string";
+                case JTokenType.Boolean:
+                    return "a boolean";
+                case JTokenType.Null:
+                    return "type null";
+                case JTokenType.Undefined:
+                    return "type undefined";
+                case JTokenType.Date:
+                    return "a date";
+                case JTokenType.Raw:
+                    return "type raw";
+                case JTokenType.Bytes:
+                    return "type bytes";
+                case JTokenType.Guid:
+                    return "a GUID";
+                case JTokenType.Uri:
+                    return "a URI";
+                case JTokenType.TimeSpan:
+                    return "a timespan";
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(jTokenType), jTokenType, null);
+            }
+        }
     }
 
     internal class Difference
     {
+        public Difference(DifferenceKind kind, JPath path, object actual, object expected) : this(kind, path)
```

Comment:
Shouldn't `actual` and `expected` be of type `string`?

Reply:
I'm also passing `int`s. This way I don't have to call `ToString`.